Gemfile

```diff
@@ -27,6 +27,8 @@ group :development, :test do
   gem 'factory_girl_rails'
   gem 'rspec-rails'
   gem 'shoulda-matchers'
+  gem 'launchy'
+  gem "capybara"
```

Comment:
Use single quotes as the other gems... And also any particular reason why we're explicitly including capybara? 🤔 

I think (and I could be wrong) the `cucumber` gem should include this already